code/game/objects/structures/girders.dm

```diff
@@ -76,6 +76,37 @@
 			user << "<span class='notice'>You slice apart the girder!</span>"
 			dismantle()
 
+	else if(istype(W, /obj/item/weapon/melee/energy))
+		var/obj/item/weapon/melee/energy/WT = W
+		if(WT.active)
+			user << "<span class='notice'>Now slicing apart the girder...</span>"
+			if(do_after(user,30))
+				if(!src) return
+				user << "<span class='notice'>You slice apart the girder!</span>"
+				dismantle()
+			else
+				user << "<span class='notice'>You need to activate the weapon to do that!</span>"
+				return
+			
+	else if(istype(W, /obj/item/weapon/melee/energy/blade))

```

Comment:
This isn't activated by the above check? I thought `istype(W, /obj/item/weapon/melee/energy)` returns true if W is a subclass of that parent as well?